### ISSUE TITLE
review

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/Pounder#44]
@gkasprow @dtcallcock what's the status of this project? Are we waiting for a review by @dtcallcock, or are we waiting or @gkasprow to finish updating the project? I've lost track...

**Comment by hartytp** [comment]
okay, I think almost all issues have been implemented. We need clarity from someone on what input power ranges we want to support, but other than that it looks like we should be able to produce another schematic RC relatively quickly from here.

**Comment by dtcallcock** [comment]
I was waiting on a new schematic after that slew of issue activity about a month ago.  Let us know if there is anything you still want to discuss though @gkasprow

**Comment by gkasprow** [comment]
It was on hold due to Booster and  Phaser. We will finish it after Xmass

**Comment by hartytp** [comment]
Nothing to do here. Let's open a new issue when there is a new RC that fixes all current issues...
